Installer/QModsInstallerScript.iss

```diff
@@ -51,49 +51,54 @@ CloseApplications=False
-Source: "..\Build\QModManager.exe"; DestDir: "{app}\Subnautica_Data\Managed"; Flags: IgnoreVersion; Check: IsSubnauticaApp
-Source: "..\Assets\QModManagerAssets.unity3d"; DestDir: "{app}\Subnautica_Data\Managed"; Flags: IgnoreVersion; Check: IsSubnauticaApp
-; Below Zero
-Source: "..\Dependencies\0Harmony.dll"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Dependencies\0Harmony-1.2.0.1.dll"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Dependencies\AssetsTools.NET.dll"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Dependencies\cldb2018.dat"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Dependencies\Mono.Cecil.dll"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Dependencies\SemVer.dll"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Dependencies\SMLHelper.dll"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Build\QModInstaller.dll"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Build\QModInstaller.xml"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Build\QModManager.exe"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
-Source: "..\Assets\QModManagerAssets.unity3d"; DestDir: "{app}\SubnauticaZero_Data\Managed"; Flags: IgnoreVersion; Check: IsBelowZeroApp
+; Files required by QModManager
+Source: "..\Dependencies\0Harmony.dll"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Dependencies\0Harmony-1.2.0.1.dll"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Dependencies\AssetsTools.NET.dll"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Dependencies\cldb.dat"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Dependencies\cldb2018.dat"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Dependencies\doorstop_config.ini"; DestDir: "{app}\QModManager"; DestName: "doorstop_config.ini.doorstop"; Flags: IgnoreVersion
+Source: "..\Dependencies\Mono.Cecil.dll"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Dependencies\SemVer.dll"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Dependencies\SMLHelper.dll"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Build\QModInstaller.dll"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Build\QModInstaller.xml"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
+Source: "..\Build\QModManager.exe"; DestDir: "{app}\QModManager"; Flags: IgnoreVersion
```

Comment:
We could put QModManager.exe in the directly in the game directory instead of the `QModManager` subdirectory so that users can see it and run it as an alternative form of patching/unpatching